### DISCUSSION TITLE
tycho version and java compatibility

### DISCUSSION THREAD

**Discussion by globavi** [eclipse-tycho/tycho#1649] [General]
Hi,

 did you state somewhere, which tycho version is compatible with which java version?
I see from release notes that from 3.0.0 java 17 is required, but shoud version < 3.0.0 already be able to run with java 17?

Best Maria

**Comment by laeubi** [comment]
Actually (beside some technical difficulties), you need to **run** Tycho:

-  Tycho 3.x -- Java 17
-  Tycho 2.x -- Java 11
- Tycho 1.x --- Java 8 ?
- Tycho 0.x  ... ?

But you should be able to **compile** with any version if you configure a toolchain.xml file and have a suitable JDK at hand.

**Reply by globavi** [reply]
thx. you meant 1.3 ->8 and 3.**0.0** -> 17? (and means 2.6.0 is not compatible with java 17 (runtime))

**Reply by laeubi** [reply]
You should be able to run 2.6.0 with Java 17 as well.
